src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+    private final int defaultRadius = 10;
+
+    public Circle() {
+        this.color = Colors.WHITE;
+        this.radius = defaultRadius;
+    }
```

Comment:
It is not a good choice to keep default parameters in the model
```suggestion
    private final int radius;

    public Circle() {
    }
```